Framework/Algorithms/src/Rebin2D.cpp

```diff
@@ -155,7 +155,7 @@ void Rebin2D::exec() {
   }
   PARALLEL_CHECK_INTERUPT_REGION
   if (useFractionalArea) {
-    outputRB->finalize(true, true);
+    outputRB->finalize(true);
```

Comment:
Can I ask why the force argument is now removed? #20953 had a fairly detailed look at this and this change was added there so I'd like to confirm we haven't broken what was done there.

Reply:
@Mantid-Matthew The one case I missed is if the input is a rebinned output and finalized. A problem is that rebinToFractionalOutput writes to the output workspace without updating the m_finalized flag to False. Rebin2D addressed this by forcing the finalize while ReflectometryTransform does not. I am happy to restore the force option as this is the only place that it is used although my preference is for rebinToFractionalOutput to leave the output in the correct state.

Reply:
@geishm-ansto Thanks for the explanation. I see what has happened. I agree that it it is better for  `rebinToFractionalOutput` to set the flag correctly. As long as the error calculation for the finalized rebinned output matches that of before then I think this is fine. Is there a test in SumSpectraTest that covers this? If not could we add one?

Reply:
The soln is not ideal as rebinToFractionalOutput is called inside a loop over (i,j) so I added a finalizeFractionalRebin that is called at the end of the loop so the flag is set once. It works.